include/real/real_data.hpp

```diff
@@ -273,7 +273,7 @@ namespace boost {
 
                     break;
                 }
-                case OPERATION::POWER: {
+                case OPERATION::INTEGER_POWER: {
                     ro.get_rhs_itr().iterate_n_times(ro.get_rhs_itr().maximum_precision());
 
                     if (ro.get_rhs_itr().get_interval().lower_bound != ro.get_rhs_itr().get_interval().upper_bound ||
```

Comment:
We are doing this check many times, maybe it makes sense to make a function in interval to ask this "isPunctual" ?

Reply:
Will do that.